include/real/real.hpp

```diff
@@ -638,12 +638,44 @@ namespace boost {
              *  @params: real_num: boost real number whose power is to be evaluated
              *  @params: exponent: power to which real_num is to be raised (needs to be an integer)
              *  @return: returns a new boost real whose value is real_num^exponent
-             *  @author: Kishan Shukla
+             *  @author: Kishan Shukla & Vikram Singh Chundawat
              */
 
-            static real power(real<T> real_num, real<T> exponent){
- 
-                return real(real_operation<T>(real_num._real_p, exponent._real_p, OPERATION::POWER));
+            static real power(real<T> real_num, real<T> power){
+                // checking whether the number is integer or not
+                static real<T> zero("0");
+                static real<T> one("1");
```

Comment:
These literals shouldn't be in the literal namespace?

Reply:
Yes they should be. But we can't use them here. Literals of real class will be defined after real class. And these variables are used inside real class.